modules/ROOT/pages/anypoint-custom-metrics-connector.adoc

```diff
@@ -1,47 +1,105 @@
-= Anypoint Custom Metrics Connector
+= Custom Metrics
 
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-The Anypoint Custom Metrics connector enables you to send operational and business metric values from a hybrid application to Anypoint Monitoring, where the metrics can be visualized in real time using a custom dashboard in advanced mode and the built-in *Custom metrics* dashboard. 
+The Anypoint Custom Metrics connector enables you to send operational and business metric values from a hybrid application to Anypoint Monitoring to visualize metrics in real time using a custom dashboard in advanced mode and the built-in *Custom metrics* dashboard. 
```

Comment:
maybe say "The Anypoint Custom Metrics connector (Custom Metrics connector) and then use the short form everywhere after this? I'm seeing inconsistencies in a few places...

Reply:
"to send...to Anypoint Monitoring...to visualize" is one "to" phrase too many? maybe "in order to" for the last one?